### PR TITLE
feat: 🎸 add result to successfully ran transactions

### PR DESCRIPTION
**Pull request by polymath-eric**
### Description

Adds a getter `result` to PolymeshTransactionBase class. this will have the result resolved from run() so it can be accessed later

### JIRA Link

✅ Closes: [DA-407](https://polymesh.atlassian.net/browse/DA-407)

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/base/PolymeshTransactionBase.ts [hunk 1]
```diff
@@ -646,6 +653,32 @@ export abstract class PolymeshTransactionBase<
     }
   }
 
+  /**
+   * returns the transaction result - this is the same value as the Promise run returns
+   * @note it is generally preferable to `await` the `Promise` returned by { @link base/PolymeshTransactionBase!PolymeshTransactionBase.run | transaction.run() } over reading this property
+   *
+   * @throws if the is { @link base/PolymeshTransactionBase!PolymeshTransactionBase.isSuccess | transaction.isSuccess } is false — be sure to check before accessing
```
Comment (VictorVicente) [inline src/base/PolymeshTransactionBase.ts:660]: Some word is missing here

Comment (polymath-eric) [reply src/base/PolymeshTransactionBase.ts]: Good catch. I think its actually left over is from "isSuccess"

**Review by VictorVicente — APPROVED** [review]

**Review by prashantasdeveloper — APPROVED** [review]

**Pull request merged**